src/main/java/com/avispl/symphony/dal/avdevices/microphone/sennheiser/tcc2/dto/DeviceWrapper.java

```diff
@@ -0,0 +1,219 @@
+/*
+ *  * Copyright (c) 2023 AVI-SPL, Inc. All Rights Reserved.
+ */
+package com.avispl.symphony.dal.avdevices.microphone.sennheiser.tcc2.dto;
+
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.time.Month;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import java.util.Locale;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import com.avispl.symphony.dal.avdevices.microphone.sennheiser.tcc2.comom.SennheiserPropertiesList;
+import com.avispl.symphony.dal.avdevices.microphone.sennheiser.tcc2.dto.audio.AudioDTO;
+import com.avispl.symphony.dal.avdevices.microphone.sennheiser.tcc2.dto.device.DeviceDTO;
+import com.avispl.symphony.dal.avdevices.microphone.sennheiser.tcc2.dto.meter.MeterDTO;
+import com.avispl.symphony.dal.avdevices.microphone.sennheiser.tcc2.dto.osc.OscDTO;
+
+/**
+ * Device Wrapper class includes device , osc , audio , meter information
+ *
+ * @author Kevin / Symphony Dev Team<br>
+ * Created on 3/14/2023
+ * @since 1.0.0
+ */
+public class DeviceWrapper {
+	private DeviceDTO device;
+	private OscDTO osc;
+	private AudioDTO audio;
+
+	@JsonProperty("m")
+	private MeterDTO meter;
+
+	/**
+	 * Retrieves {@link #device}
+	 *
+	 * @return value of {@link #device}
+	 */
+	public DeviceDTO getDevice() {
+		return device;
+	}
+
+	/**
+	 * Sets {@link #device} value
+	 *
+	 * @param device new value of {@link #device}
+	 */
+	public void setDevice(DeviceDTO device) {
+		this.device = device;
+	}
+
+	/**
+	 * Retrieves {@code {@link #osc}}
+	 *
+	 * @return value of {@link #osc}
+	 */
+	public OscDTO getOsc() {
+		return osc;
+	}
+
+	/**
+	 * Sets {@code osc}
+	 *
+	 * @param osc the {@code com.avispl.symphony.dal.avdevices.microphone.sennheiser.tcc2.dto.osc.OscDTO} field
+	 */
+	public void setOsc(OscDTO osc) {
+		this.osc = osc;
+	}
+
+	/**
+	 * Retrieves {@code {@link #audio}}
+	 *
+	 * @return value of {@link #audio}
+	 */
+	public AudioDTO getAudio() {
+		return audio;
+	}
+
+	/**
+	 * Sets {@code audio}
+	 *
+	 * @param audio the {@code com.avispl.symphony.dal.avdevices.microphone.sennheiser.tcc2.dto.audio.AudioDTO} field
+	 */
+	public void setAudio(AudioDTO audio) {
+		this.audio = audio;
+	}
+
+	/**
+	 * Retrieves {@code {@link #meter }}
+	 *
+	 * @return value of {@link #meter}
+	 */
+	public MeterDTO getMeter() {
+		return meter;
+	}
+
+	/**
+	 * Sets {@code beam}
+	 *
+	 * @param meter the {@code com.avispl.symphony.dal.avdevices.microphone.sennheiser.tcc2.dto.meterDTO.BeamDTO} field
+	 */
+	public void setMeter(MeterDTO meter) {
+		this.meter = meter;
+	}
+
+	/**
+	 * get Object value by command Enum
+	 *
+	 * @param commandEnum command Enum value
+	 * @return value of object
+	 */
+	public String getObjectByName(SennheiserPropertiesList commandEnum) {
+		switch (commandEnum) {
+			case SERIAL_NUMBER:
+				return device.getIdentity().getSerial();
```

Comment:
please check null before getting every single instance of the data, because it will cause the null pointer exception
ForExample: Optional.ofNullable(device.getIdentity()).map(IdentityDTO::getSerial).orElse(SennheiserConstant.NONE);